docs/src/data/hid.js

```diff
@@ -5896,7 +5896,7 @@ export default [
     ],
     documentation: "https://usb.org/sites/default/files/hut1_2.pdf#page=137",
     os: {
-      windows: null,
+      windows: false,
```

Comment:
`C_PREV` in general works for me, specifically for Spotify. Is it possible that it is a foobar-specific issue?

Reply:
It may be software-specific. Did you use the web client or the application for Spotify? It may be good to test it on both fronts.

Reply:
Sorry, I just noticed this. I also tested with Spotify web player and Youtube and they both work well with `C_PREV` as well.

Reply:
If it works for you now, I think it's fine to list `C_PREV` as functional in the docs. If you don't mind submitting a commit suggestion, I'll swiftly apply it.